### PR TITLE
[Display Layout] Allow duplication and changing types

### LINKED ISSUE #2973: [Display Layout] A better workflow for adding plots into display layout
At the moment, users have to create an overlay or stacked plot to add endpoints to display layouts. 
A suggested workflow is - users are able to drag endpoints directly into Display Layout and then choose wether they want an alphanumeric, plot or table

### PR DESCRIPTION
**Pull request by deeptailor**
This PR adds the following functionalities to display layout:
1. Allow duplication of objects, and support multiselect
2. Allow users to change view types:
-  - From alpha numerics to overlay plot, stacked plot and table.
-  - From Tables to alpha numeric, stacked plot or overlay plot
-  - From Overlay plots to stacked plot (multi select)
-  - From single overlay plots to table stacked plot and alpha numeric

This PR fixes #2973 

@charlesh88 - Needs an icon for alpha numerics

### CODE REVIEW

**Comment by charlesh88** [comment]
@deeptailor Just pushed new icon and config changes to this branch.
### Changes
- Add font files and icomoon JSON file;
- Mods to constants.scss and glyphs files to add new glyph;
- Modify toolbar to use new icon;
- Changed tooltip text;
- Change placement of view switch button in toolbar layout;

### Testing Notes
1. Create a new Display Layout.
1. Add a telemetry element (sine wave/event generator/etc.). It will be selected.
1. Verify that the first icon in the toolbar is the "View switch" icon button menu:
Screen Shot 2020-06-04 at 7 30 12 PM
1. Click the button and change the view to a plot or table;
1. Click the button again and verify that "Alphanumeric" is available as an option in the menu, and that the menu entry uses the proper glyph:
Screen Shot 2020-06-04 at 7 32 15 PM

File: src/plugins/displayLayout/components/DisplayLayout.vue [hunk 1]
```diff
@@ -515,6 +526,190 @@ export default {
...
+                this.trackItem(copy);
+                this.layoutItems.push(copy);
+            });
+
+            this.$nextTick(() => {
+                this.openmct.objects.mutate(this.internalDomainObject, "configuration.items", this.layoutItems);
+                this.openmct.objects.mutate(this.internalDomainObject, "configuration.objectStyles", objectStyles);
+                this.$el.click(); //clear selection;
+
+                this.$nextTick(() => {
```
Comment (davetsay) [inline src/plugins/displayLayout/components/DisplayLayout.vue:632]: Dang. Does that this.$el.click() make this nested $nextTick required? Craziness.

File: src/plugins/displayLayout/components/DisplayLayout.vue [hunk 2]
```diff
@@ -365,7 +367,8 @@ export default {
                 let count = this.telemetryViewMap[keyString] || 0;
                 this.telemetryViewMap[keyString] = ++count;
             } else if (item.type === "subobject-view") {
-                this.objectViewMap[keyString] = true;
+                let count = this.objectViewMap[keyString] || 0;
+                this.objectViewMap[keyString] = ++count;
```
[the remark this answers is not in the document]
Comment (deeptailor) [reply src/plugins/displayLayout/components/DisplayLayout.vue:371]: they account for different items, objectViewMap counts the number of object-views we have and telemetryViewMap counts the number of telemetry-views we have

[the remark this answers is not in the document]
Comment (davetsay) [reply src/plugins/displayLayout/components/DisplayLayout.vue:371]: I see. So here we are looking at a single `count` variable which could be derived from either object views or telemetry views. Down in lines 391-392 we need to have two separate count variables derived from object views and telemetry views hence `objectViewCount` and `telemetryViewCount`.

Thanks for the explanation. Makes sense.

File: src/plugins/displayLayout/components/DisplayLayout.vue [hunk 3]
```diff
@@ -413,7 +424,7 @@ export default {
         addChild(child) {
             let identifier = this.openmct.objects.makeKeyString(child.identifier);
```
[the remark this answers is not in the document]
Comment (deeptailor) [reply src/plugins/displayLayout/components/DisplayLayout.vue]: Done, changed it to keystring where it made sense

File: src/plugins/displayLayout/components/DisplayLayout.vue [hunk 4]
```diff
@@ -515,6 +526,190 @@ export default {
...
+                    key: uuid(),
+                    namespace: domainObject.identifier.namespace
+                },
+                type = this.openmct.types.get(viewType),
+                parentKeyString = this.openmct.objects.makeKeyString(this.internalDomainObject.identifier),
+                objectName = nameExtension ? `${domainObject.name}-${nameExtension}` : domainObject.name,
+                object = {};
+
+            if (model) {
+                object = this.createDeepCopy(model);
```
[the remark this answers is not in the document]
Comment (deeptailor) [reply src/plugins/displayLayout/components/DisplayLayout.vue]: I removed the helper function and used _.cloneDeep, as we use it in multiple other places in display Layout

File: src/plugins/displayLayout/components/DisplayLayout.vue (hunk 1 above)
Comment (deeptailor) [reply src/plugins/displayLayout/components/DisplayLayout.vue]: I thought it was needed to be safe, but I removed it and did some testing

**Comment by deeptailor** [comment]
## Author Checklist

1. Changes address original issue? Yes
2. Unit tests included and/or updated with changes? No, need to backlog since we have a few more changes coming to display layout
3. Command line build passes? Yes
4. Changes have been smoke-tested? Yes
5. Testing instructions included? Yes

File: src/plugins/displayLayout/components/DisplayLayout.vue (hunk 1 above)
Comment (davetsay) [reply src/plugins/displayLayout/components/DisplayLayout.vue]: Okay. Nice.

**Pull request merged**